AppAbyss.WebApi/Program.cs

```diff
@@ -7,13 +12,21 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
+{
+    options
+        .UseSqlServer(connectionString, x => x.MigrationsAssembly("AppAbyss.Repository"))
+        .UseLazyLoadingProxies();
+});
+
 var app = builder.Build();
 
 // Configure the HTTP request pipeline.
 if (app.Environment.IsDevelopment())
 {
     app.UseSwagger();
     app.UseSwaggerUI();
+    builder.Services.AddDatabaseDeveloperPageExceptionFilter();
```

Comment:
This line throws an InvalidOperationException.

Reply:
I have fixed it.